docs/index.yml

```diff
@@ -1,7 +1,7 @@
 ### YamlMime:Landing
 
 title: Office Add-ins documentation # < 60 chars
-summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, and iPad. # < 160 chars
+summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, and iPad. To build your first add-in within a few minutes, check out the Quick starts. To take a deeper dive into add-in development, explore the Tutorials. For an interactive learning experience, try the Microsoft Learn Office Add-in modules, like Introduction to Office client customization with add-ins and Extend Office clients with Office add-ins. # < 160 chars
```

Comment:
The comment at the end seems to imply that this value has to be fewer than 160 characters. Is that not true?

Reply:
The original was already longer than 160 - maybe we consider this a suggestion, not a hard-and-fast rule...
Landing page documentation: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-landing-page?branch=master

Reply:
Thank you for this link, Elizabeth! I think this paragraph is too long as is (it's inaccessible and hard to parse), so I only want to add my new content if there's a way to break it into two paragraphs. Based on the documentation + my testing last week, I don't think that's possible. I also originally envisioned including a link to the Learn module here, and that doesn't seem possible in the summary field either.